src/main/java/com/faforever/client/api/MockFafApiAccessor.java

```diff
@@ -160,22 +160,22 @@ public Mod getMod(String uid) {
   }
 
   @Override
-  public List<Game> getNewestReplays(int count) {
+  public List<Game> getNewestReplays(int count,int page) {
```

Comment:
Please enable the auto-formatting in your commit settings, or if you have done it already, reformat this file :)